samples/net/prometheus/README.rst

```diff
@@ -0,0 +1,87 @@
+Zephyr Prometheus Example
+=========================
```

Comment:
Use `.. zephyr:code-sample::` directive and don't forget to flag `prometheus` as part of the "`relevant_apis`"

Reply:
Updated with following block:

```rst
.. zephyr-app-commands::
   :zephyr-app: samples/net/prometheus
   :board: <board to use>
   :conf: <config file to use>
   :goals: build
   :compact:
```